### ISSUE TITLE
[BUG] When set to display aliases instead of usernames, reactions are displayed as coming from usernames

### ISSUE THREAD

**Issue by itsthejoker** [RocketChat/Rocket.Chat#9759]
### Description:

On our server, `Set a User Name to Alias in Message` is set to True. This setting does not affect emoji reacts, and it seems like it should.

### Server Setup Information:

* Version of Rocket.Chat Server: 0.61.0
* Operating System: All
* Deployment Method(snap/docker/tar/etc): Docker
* Number of Running Instances: 1
* DB Replicaset Oplog: Enabled
* Node Version: v8.9.3

### Steps to Reproduce:

1. Set `Set a User Name to Alias in Message` to True
2. Post a message
3. Have someone else react to that message

### Expected behavior:

I expect to see that Pejman posted a react and not `@thepageman`.

### Actual behavior:

![usernames when set to display aliases](https://user-images.githubusercontent.com/5179553/36345309-e3cbf9d6-13f5-11e8-913a-5c72337bc338.PNG)

**Comment by sumedh123** [comment]
@rafaelks Any opinions on this issue? Can I work on this one?

**Comment by rafaelks** [comment]
@sumedh123 Yes, you can work on this one since is not assigned to anyone yet. Go for it. 👍

**Comment by tomasz-kuzemczak** [comment]
Any progress on this? :) Just noticed this on our instance of Rocket Chat and would love to see this fixed.

**Comment by hyzteric** [comment]
Yeah I notice that when I start the RocketChat client for the first time, I see the Alias and not the user name.
But any further reaction will show the user name and not the Alias as expected. Very annoying as user name here is an ldap id... and Alias is our name...
